samples/base/CMakeLists.txt

```diff
@@ -1,22 +1,73 @@
-cmake_minimum_required(VERSION 3.1)
-
-# determine the build type
-if(NOT CMAKE_BUILD_TYPE)
```

Comment:
For what reason do you have remove that? You may need this for the Android Build